docs/user/bots.md

```diff
@@ -2154,7 +2153,7 @@ correct mapping of the columns:
 
 **`feedname`**
 
-(optional, string) Name of the Shadowserver report, see list below for possible values.
+(optional, string) Name of the Shadowserver report. The value for each report type can be found in the schema `feed_name` field.
```

Comment:
For example using `curl https://interchange.shadowserver.org/intelmq/v1/schema | jq .[].feed_name`?